addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1494,8 +1494,17 @@ options.registry.WebsiteFormFieldModel = DisableOverlayButtonOption.extend({
 // Disable delete button for model required fields
 options.registry.WebsiteFormFieldRequired = DisableOverlayButtonOption.extend({
     start: function () {
-        this.disableButton('remove', _t('This field is mandatory for this Action. You cannot remove it.'));
+        this.disableButton("remove",
+_t("This field is mandatory for this action. You cannot remove it. Try hiding it with the 'Visibility' option instead and add it a default value."));
```

Comment:
I know this line is too long but if I split it into two, I got this when I export translations:
![image](https://user-images.githubusercontent.com/78849981/237042713-0624d9c3-fe54-4d44-bf38-72865a9a7aa5.png)

The string is also split in the po file which seems not a good thing...

Reply:
It should be perfectly fine to line return it.
Eg if you do:
```js
const myVar = _t(
    "The barcode nomenclature setting is not configured. " +
    "Make sure to configure it on your Point of Sale configuration settings"
);
```

You will have something like this in `.po`:
```
#. module: point_of_sale
#. odoo-javascript
#: code:addons/point_of_sale/static/src/js/Chrome.js:0
#, python-format
msgid ""
"The barcode nomenclature setting is not configured. Make sure to configure "
"it on your Point of Sale configuration settings"
msgstr ""
"Le paramètre de nomenclature des codes-barres n'est pas configuré. Assurez-"
"vous de le configurer dans les paramètres de votre Point de Vente"
``

You will have something like this in [transifex](https://app.transifex.com/odoo/odoo-16/translate/#fr/point_of_sale/446905189?q=text%3A'The+barcode+nomenclature+setting'):
![image](https://github.com/odoo/odoo/assets/30048408/0b5ff4c5-dc20-48aa-a2c7-e43b5866e714)

The line return you see in the `.po`/`.pot` file is just because those files are auto line returning at 80 char, regardless of the initial source string size.

So if you export a translation term which is a single sentence 200 char long, it will be split in 3 lines in the `.po(t)` file(s).

Reply:
Thanks RDE :blush: